doc/examples/reconst_ivim.py

```diff
@@ -0,0 +1,257 @@
+"""
+============================================================
+Intravoxel incoherent motion
+============================================================
+The intravoxel incoherent motion (IVIM) model describes diffusion
+and perfusion in the signal acquired with a diffusion MRI sequence
+that contains multiple low b-values. The IVIM model can be understood
+as an adaptation of the work of Stejskal and Tanner [Stejskal65]_
+in biological tissue, and was proposed by Le Bihan [LeBihan84]_.
+The model posits that two compartments exist: a slow moving compartment,
+where particles diffuse in a Brownian fashion as a consequence of thermal
+energy, and a fast moving compartment (the vascular compartment), where
+blood moves as a consequence of a pressure gradient. In this second
+compartment, a pseudo diffusion term $\mathbf{D^*}$ is introduced that
+describes the displacement of the blood elements in an assumed randomly
+laid out vascular network, at the macroscopic level. For the perfusion
+to have a physiological meaning, one expects that $\mathbf{D^*}$ is
+greater than $\mathbf{D}$.
```

Comment:
You are mentioning that $\mathbf{D^*} $is greater than $\mathbf{D}$ but you never defined $\mathbf{D}$. Please do this when describing the first compartment of the model.